fedimint-server/src/consensus/transaction.rs

```diff
@@ -97,7 +97,7 @@ impl FundingVerifier {
     }
 
     pub fn verify_funding(self) -> Result<(), TransactionError> {
-        if self.input_amount == (self.output_amount + self.fee_amount) {
+        if self.input_amount >= (self.output_amount + self.fee_amount) {
```

Comment:
Technically a consensus break.

Reply:
The second commit was only for testing and has been removed.